app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(string: str) -> None:
+    names = string.split(" ")
+    if names[0] == "cp":
+        if names[1] != names[2]:
```

Comment:
By using unpacking, you make it clear that `names[1]` is the source file, and `names[2]` is the destination file, which can improve the code's readability.But before unpacking you should check for length